flixel/FlxCamera.hx

```diff
@@ -1256,10 +1255,8 @@ class FlxCamera extends FlxBasic
 			{
 				return;
 			}
-			// This is temporal fix for camera's color
```

Comment:
Wow, this code was _old_! https://github.com/HaxeFlixel/flixel/commit/227a4647b8b1f38b11eacc98bf1be57203c1381e#diff-9ef1aac582dab97c9bde1a8fb693a1b7R845